### DISCUSSION TITLE
Creation of new folders when adding collection item

### DISCUSSION THREAD

**Discussion by M-GRV** [directus/directus#18363] [Feature Requests]
### Summary

Currently you can only select a media folder that already exists. It would be great to allow creation of a new folder for a new entry made in the collection.

This will allow us to group new entries media content into their own respective folder. 

### Motivation

If I am adding a new model year to a model, I'd like for all the new media for that model year to be attached within its own folder.

So for example I create a collection called models. I also create a main media folder called models. 

Now If I create multiple entries, all the media files for those entries are lumped together in the model folder.

It would be nice if I could have the option to set a new folder to be created using one of the string fields of the entry being made.

### Detailed Design

![image](https://user-images.githubusercontent.com/130182174/235188909-1d2fdcb1-c7fe-4532-9bcb-6f907f54e96b.png)
Here is what it looks like today. 

We could add a new option right above or below the file library root option to create new folder. 

### Must Have:
- A - Creation of new folders when adding a collection item

### Drawbacks

Nothing I can see at the moment for a drawback to this. 

### Alternatives

You have to currently do this manually before adding a new collection entry. When adding thousands of items, it can take a lot of time to manually create and set a folder. But you still cant set a folder for each entry. 

### Adoption Strategy

Not a breaking change.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by lupus78** [comment]
I'm also missing this feature badly. IMHO the current common for folder for each content item is very hard to maintain. The benefit would be that content creators can select the already uploaded image, but it's very hard to find in a folder with hundreds of images.

What I would love to see is an option to create a subfolder for each item, something like blog/[id] and than all the files would be uploaded there.

**Reply by M-GRV** [reply]
thats exactly how I was thinking. We will have to re-submit this idea and get 15 votes for it to be considered.

**Reply by lupus78** [reply]
I'm fully behind. Can't we just reopen this one? @rijkvanzanten

**Reply by dafrica1** [reply]
I'd also like this to be reopened, the process of searching for files becomes a lot less cumbersome with such a structure.
